### PR TITLE
chore: use vue-demi for backward compat with vue2

### PR DESCRIPTION
**Pull request by messenjer**
Use vue-demi - https://github.com/MaxLeiter/sortablejs-vue3/issues/69

### CODE REVIEW

**Review by MaxLeiter — CHANGES_REQUESTED** [review]
Can you adjust the README to mention vue 2 compatibility?

File: package.json [hunk 1]
```diff
@@ -36,11 +36,17 @@
   },
   "dependencies": {
     "sortablejs": "^1.15.0",
-    "vue": "^3.2.37"
+    "vue": "^3.2.37",
+    "vue-demi": "latest"
   },
```
Comment (MaxLeiter) [inline package.json:41]: Can you specify the version?

File: package.json [hunk 2]
```diff
@@ -50,6 +56,7 @@
     "terser": "5.16.1",
     "typescript": "4.9.4",
     "vite": "4.0.4",
+    "vue": "^3.2.37",
```
Comment (MaxLeiter) [inline package.json:59]: We shouldnt need vue in the dev deps if its also in the regular deps, afaik

**Review by MaxLeiter — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (messenjer) [reply package.json:41]: done

File: package.json (hunk 2 above)
Comment (messenjer) [reply package.json]: done

**Comment by messenjer** [comment]
> Can you adjust the README to mention vue 2 compatibility?

done, also

**Review by MaxLeiter — APPROVED** [review]

**Comment by MaxLeiter** [comment]
Thanks!

**Pull request merged**